templates/actors/config/concentration-config.hbs

```diff
@@ -0,0 +1,24 @@
+<section class="flexcol">
+    {{!-- Concentration --}}
+    <fieldset class="card">
+        <legend>{{ localize "DND5E.Concentration" }}</legend>
+        {{ formField fields.ability value=data.ability options=abilityOptions localize=true }}
+        {{ formField fields.bonuses.fields.save value=data.bonuses.save localize=true }}
+        {{ formField fields.limit value=data.limit placeholder="1" localize=true }}
+        <hr>
+        {{ formField fields.roll.fields.mode value=data.roll.mode options=advantageModeOptions localize=true }}
+        {{ formField fields.roll.fields.min value=data.roll.min type="number" localize=true }}
+        {{ formField fields.roll.fields.max value=data.roll.max type="number" localize=true }}
```

Comment:
I think either we need a separate fieldset with a legend of 'Roll'/'Save', or we need to change the labels on these to 'Minimum Roll' and 'Maximum Roll', as just 'Minimum' and 'Maximum' is maybe a little unclear.

We might also consider a split-group with 'Roll': 'Minimum' 'Maximum' too. Though if we're expecting these fields to be filled with complicated roll expressions then perhaps they're better on their own lines.